src/sentry/integrations/gitlab/client.py

```diff
@@ -28,6 +29,7 @@ class GitLabApiClientPath(object):
     project_hooks = u"/projects/{project}/hooks"
     project_hook = u"/projects/{project}/hooks/{hook_id}"
     user = u"/user"
+    file = u"/projects/{id}/repository/files/{path}"
```

Comment:
@ceorourke we should keep using `project` here I think to be consistent with other routes.